syntax = "proto3";

message UserLogin
{
  int32 nt_type = 1;
  int64 user_id = 2;
  string name   = 3;
}
